src/main/java/com/softserve/kickscootertrip/service/TripService.java

```diff
@@ -0,0 +1,87 @@
+package com.softserve.kickscootertrip.service;
+
+import com.softserve.kickscootertrip.model.dto.TripDto;
+import com.softserve.kickscootertrip.model.dto.UserDto;
+import com.softserve.kickscootertrip.model.entity.TripEntity;
+import com.softserve.kickscootertrip.repository.TripRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.core.convert.ConversionService;
+import org.springframework.kafka.annotation.KafkaListener;
+import org.springframework.kafka.core.KafkaTemplate;
+import org.springframework.stereotype.Service;
+
+import java.io.IOException;
+import java.util.Date;
+import java.util.List;
+import java.util.UUID;
+
+@Service
+@RequiredArgsConstructor
+public class TripService {
+
+    private static final double RADIUS_EARTH = 6371008.67;
+    private static final String SALES = "TripForSales";
+    private static final String PAYMENT = "TripForPayment";
+    private final TripRepository tripRepository;
+    private ConversionService convert;
+
+    private final KafkaTemplate<String, TripDto> kafkaTemplate;
+
+    public TripDto sendTripToPayment(UUID tripId){
+        TripEntity tripEntity = tripRepository.findByTripId(tripId);
+        TripDto tripDto = convert.convert(tripEntity, TripDto.class);
+        kafkaTemplate.send(PAYMENT, tripDto);
+        return tripDto;
+    }
+
+    public TripDto findUserTrips(UUID id) {
+        TripDto tripDto = new TripDto();
+        List<TripEntity> tripEntities = tripRepository.findByUserId(id);
+        if (tripEntities.size() != 0) {
+            tripDto.setUserId(id);
+            int tripsNumber = 0;
+            for (TripEntity tripEntity : tripEntities) {
+                tripsNumber++;
```

Comment:
tripsNumber = tripEntities.size()
Why need increment from 0 tripsNumber ?
Do you have special case when tripsNumber is not increment inside for loop?